MapboxNavigationUI/NavigationMapView.swift

```diff
@@ -1,10 +1,14 @@
 import Foundation
+import MapboxDirections
 
-public class NavigationMapView: MGLMapView {
+open class NavigationMapView: MGLMapView {
     
     var navigationMapDelegate: NavigationMapViewDelegate?
     
-    public override func locationManager(_ manager: CLLocationManager!, didUpdateLocations locations: [Any]!) {
+    let cap = NSValue(mglLineCap: .round)
+    let join = NSValue(mglLineJoin: .round)
```

Comment:
A boxed line join generally isn’t interesting enough to split out into a separate variable.